src/views/MethodologyPage.vue

```diff
@@ -1,17 +1,86 @@
 <template>
-  <div>
-    <div>This is the Methodology page</div>
-  </div>
+  <section class="w-5/6 mx-auto py-20 text-center">
+    <div class="pt-20">
+      <h1 class="uppercase font-bold text-5xl">Methodology</h1>
+      <p class="pt-4 lg:w-3/4 mx-auto">A prioritized list of MITRE ATT&amp;CK techniques should be actionable and driven
+        by threat intelligence. The Center’s methodology is composed of three different components - Actionability,
+        Choke
+        Point, and Prevalence. Algorithms for each component were created to determine a technique’s weight within a
+        specific component, and then each component weight is combined to give an overall weight. </p>
+    </div>
+    <div class="container-row">
+      <div class="lg:w-1/4">
+        <div class="w-full h-auto lg:border-0 border-[1px] border-ctid-black mb-4">
+          <MethodologySidebar :methodology-list="methodologyList" :active-item-id="activeItemId"
+            @set-active-index="(i) => activeItemId = i" />
+        </div>
+      </div>
+      <div class="lg:w-3/4">
+        <div class="calculator-details">
+          <div class="container-header">
+            <h2 class="text-left">
+              <span>
+                {{ methodologyList[activeItemId].name }}
+              </span>
+            </h2>
+          </div>
+          <div v-for="(item, i) of methodologyList" :key="i">
```

Comment:
The reason I have a loop is so that the mathjax gets rendered - if I do it the way you suggested, then the mathjax for the other two sections doesn't render when the user clicks on them

So, I'm including all the section code when building the page and hiding the unneeded sections which isn't ideal but it does make the mathjax work